schema/components.json

```diff
@@ -171,7 +171,8 @@
                         "usb-3-micro-b",
                         "dc-terminal",
                         "saf-d-grid",
-                        "hardwired"
+                        "hardwired",
```

Comment:
This file cannot be modified. These are what netbox has upstream by default. **Please remove your changes here and any use of these proposed new ones need to be changed**. PSU types should either reflect one of the already pre-defined names, or `other` if there is not a current type available

If you want a new type added, you will need to be propose a change to the upstream netbox repo. Once added in that repo and pushed to master, we can add them here as well

Reply:
I was not aware the file can't be changed - I'm new to netbox...

In this case dc-terminal is fine for me.

Reply:
Perfect, go ahead and revert the changes here, then update your files accordingly. Thank you!